lib/src/tooltip_widget.dart

```diff
@@ -196,6 +196,40 @@ class _ToolTipWidgetState extends State<ToolTipWidget> {
     final arrowWidth = 18.0;
     final arrowHeight = 9.0;
 
+    final arrowWidget = !widget.showArrow!
+        ? []
+        : [
+            Positioned(
+              left: _getLeft() == null
+                  ? null
+                  : max<double>(
+                      (widget.position!.getCenter() -
+                          (arrowWidth / 2) -
+                          (_getLeft() ?? 0)),
+                      4),
+              right: _getLeft() == null
+                  ? max<double>(
```

Comment:
added minimum offset here

Reply:
@CCFrankUrbainsky This is good. However, it is still a static value (4) so if the border radius is changed(I am assuming it can be changed), the issue might happen. Can we use that border radius to offset the arrow? cc: @DevarshRanpara